Add initial functions for openIaaS

julienmorot-ct opened the pull request.

julienmorot-ct committed:
add initial functions to generate openiaas datasources for cloud temple terraform provider

julienmorot-ct committed:
Add storage repo

KChapron committed:
Here's my updates of the "normal" expectations of the refresh token flow

Theoooooo committed:
Update policies call

Theoooooo reviewed (approved):
Only the functions should be added inside the pull request, the other modifications have been merged already

It should solved itself during the pull request without conflict

aelttil committed:
feat: Add Order API client and consumption retrieval script

- Add Order class in libs/ShivaApi.py with get_consumptions() method
- Support date format YYYY-MM-DD with automatic conversion to timestamps
- Add tests/get_consumption.py script to fetch and display consumption data
- Display SKU usage with descriptions in French
